### ISSUE TITLE
Training Loss Plateau Around 3.5–3.7 — Is This Expected? (Sanskrit Fine-tuning)

### ISSUE THREAD

**Issue by kartikeykushwaha14** [k2-fsa/OmniVoice#126]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

Hi,

I’m currently fine-tuning the model on a Sanskrit language dataset and noticed that both training and evaluation loss seem to plateau in a narrow range (~3.5–3.7) even after extended training.

Here are some logs from my recent run:

Steps    Train Loss    Eval Loss
30000    3.5773        3.7257
31000    3.5478        3.7350
32000    3.5320        3.7279
33000    3.4983        3.7155
34000    3.5166        3.7404
35000    3.5481        3.7437
36000    3.5031        3.7470
37000    3.4934        3.7167
38000    3.5775        3.7333
39000    3.5550        3.7121
40000    3.4737        3.7037
41000    3.5266        3.7189
42000    3.5397        3.7220
43000    3.4966        3.7030
44000    3.5075        3.7072
45000    3.5000        3.7043
46000    3.5044        3.7315
47000    3.5238        3.7167
48000    3.5222        3.6854
49000    3.5362        3.7178
50000    3.5325        3.7091

 Observations :
Training loss fluctuates between 3.47–3.57
Evaluation loss stays around 3.68–3.74
No clear downward trend after ~30k steps
Train and eval losses are close (no obvious overfitting)

Dataset size: 72 h
learning rate : 2e-5

And i use your training provided script train.py by doing 
"init_from_checkpoint":"By giving here own model path",
 

❓ Questions
Is this kind of loss plateau expected when fine-tuning on Sanskrit data?

Any guidance would be really helpful. Thanks!

**Comment by zhu-han** [comment]
It seems reasonable to me. The loss does not always keep decreasing. You should pay closer attention to downstream performance during fine-tuning (e.g., monitoring WER or directly listening to the generated audio). Besides, note that fine-tuning may not always yield better performance for your language. It's just fitting your fine-tuning data better.

**Comment by kartikeykushwaha14** [comment]
Thanks for taking the time to help. Really appreciate it!
